src/st/dre/dre.h

```diff
@@ -134,7 +134,7 @@ extern u8 D_801813FC;
 
 extern LayoutEntity* g_LayoutObjHorizontal;
 extern LayoutEntity* g_LayoutObjVertical;
-extern s8 g_LayoutObjPosHorizontal;
+extern u8 g_LayoutObjPosHorizontal;
```

Comment:
This was a pain and I wonder if it thwarted previous attempts to decompile the horizontal entity creation methods here and in `mad`.